src/Operation/Aggregate.php

```diff
@@ -287,9 +287,10 @@ public function execute(Server $server)
      * Create the aggregate command.
      *
      * @param Server  $server
+     * @param boolean $hasOutStage
      * @return Command
      */
-    private function createCommand(Server $server)
+    private function createCommand(Server $server, $hasOutStage)
```

Comment:
Instead of passing in ``$hasOutStage`` as an argument, would not be nicer to have this method run ``$hasOutStage = \MongoDB\is_last_pipeline_operator_out($this->pipeline);`` itself? I realise it's a ``private`` function, but IMO passing around unnecessary arguments is bad form.